tools/project_manager/automodify.php

```diff
@@ -252,19 +252,19 @@ function ensure_project_blurb($project)
     if ($state == $round->project_complete_state) {
         // The project is ready to exit this round.
 
-        if ($round->round_number < MAX_NUM_PAGE_EDITING_ROUNDS) {
+        if ($round->round_number < Rounds::get_last()->round_number) {
             // It goes to the next round.
             $next_round = get_Round_for_round_number(1 + $round->round_number);
             $new_state = $next_round->project_waiting_state;
-        } elseif ($round->round_number == MAX_NUM_PAGE_EDITING_ROUNDS) {
+        } elseif ($round->round_number == Rounds::get_last()->round_number) {
```

Comment:
This could now be shortened to `($round == Rounds::get_last())`

Reply:
We could, but I'd like to leave this as-is so it's parallel with the `<` comparison in the first `if()` block. Unfortunately it's not possible to make custom comparison functions in PHP objects like you can do in Python which would let us do `$round1 < $round2`.

Reply:
A third option, although in this case it's probably more obscure, is to switch on the spaceship operator
```
switch ($round->round_number <=> Rounds::get_last()->round_number) {
 case -1: // increment to next round
 case 0:  // go to postprocessing
 case 1: // should be impossible
}
```